source/crud/tests/v2/unacknowledged-bulkWrite-delete-hint.yml

```diff
@@ -0,0 +1,94 @@
+runOn:
+  - { minServerVersion: "4.3.4" }
+
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+  - {_id: 3, x: 33}
+  - {_id: 4, x: 44}
+
+collection_name: &collection_name 'BulkWrite_delete_hint'
+
+tests:
+  -
+    description: "Unacknowledged bulkWrite deleteOne with hints"
```

Comment:
nit: this might be more of a refactor than its worth, but should these descriptions actually describe the behavior? e.g. `Unacknowledged bulkWrite deleteOne with hints succeeds on server >4.4`. I've been noticing this recently (the EJSON tests for "All BSON Values" say "Document with multiple values" for both the non-deprecated and deprecated files), and think it can lead to confusing test output.

Reply:
No I totally agree with this! Good point.